CallAutomation_AppointmentReminder/CallAutomation_AppointmentReminder/Program.cs

```diff
@@ -22,15 +22,11 @@
 // Api to initiate out bound call
 app.MapPost("/api/call", async (CallAutomationClient callAutomationClient, IOptions<CallConfiguration> callConfiguration, ILogger<Program> logger) =>
 {
-    var source = new CallSource(new CommunicationUserIdentifier(sourceIdentity))
-    {
-        CallerId = new PhoneNumberIdentifier(callConfiguration.Value.SourcePhoneNumber)
-    };
+    var CallerId = new PhoneNumberIdentifier(callConfiguration.Value.SourcePhoneNumber);
     var target = new PhoneNumberIdentifier(callConfiguration.Value.TargetPhoneNumber);
```

Comment:
I need this number can be sent from api query param with ';' separated, and if not available from there, then use this number from config.